web/README.md

```diff
@@ -0,0 +1,418 @@
+}
+```
+
+### Session
+
+(1) `/session` is a GET API which returns the schema conversion state in json format. It also create a file with suffix `.session.json` in the `frontend/` folder.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Session file info with driver name.
+
+Example
+
+```json
+{
+  "Driver": "postgres",
+  "FilePath": "/path/to/sessionFile",
+  "FileName": "sessionFile.session.json",
+  "CreatedAt": "2021-01-01T00:00:00.000000+00:00"
+}
+```
+
+(2) `/session/resume` is a POST API which can be used to resume the previous session, it requires the session file which can be generated using the above API. It takes following as request body.
+
+#### Method
+
+`POST`
+
+#### Request body
+
+Provide driver and session file path.
+
+Example
+
+```json
+{
+  "driver": "postgres",
+  "path": "/path/to/sessionFile",
+  "fileName": "sessionFile.session.json"
+}
+```
+
+#### Response body
+
+No response body is returned.
+
+### Summary
+
+`/summary` is a GET API which returns table wise summary of the conversion.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Table wise summary of conversion.
+
+Example
+
+```json
+{
+  "Albums": "Note\n1) Some columns will consume more storage in Spanner e.g. for column 'AlbumId', source DB type smallint(6) is mapped to Spanner type int64.\n\n"
+}
+```
+
+### Overview
+
+`/overview` is a GET API which returns the overview of the conversion.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Overall summary of conversion in string format.
+
+### Conversion
+
+`/conversion` is a GET API which returns table wise rate of conversion which is encoded in color values.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Table wise rate of conversion encoded in color values.
+
+Example
+
+```json
+{
+  "Albums": "GREEN",
+  "Singers": "BLUE",
+  "Userinfo": "RED"
+}
+```
+
+### Typemap
+
+(1) `/typemap` is a GET API which returns the source type to list of spanner type map, which can be used for manual type maping from the UI.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Map from source type to list of possible spanner type and issue.
+
+Example
+
+```json
+{
+  "smallint": [
+    {
+      "T": "INT64",
+      "Brief": "Some columns will consume more storage in Spanner"
+    },
+    {
+      "T": "STRING",
+      "Brief": "Some columns will consume more storage in Spanner"
+    }
+  ],
+  "varchar": [
+    {
+      "T": "BYTES",
+      "Brief": ""
+    },
+    {
+      "T": "STRING",
+      "Brief": ""
+    }
+  ]
+}
+```
+
+(2) `/typemap/global` is a POST API which converts from source type map to desired spanner typemap globally (i.e, it affects all the tables). It takes a map from source type to spanner type in json format.
```

Comment:
This explanation is a little confusing -- can you rephrase?

I'm assuming it is the API call to actually customize schema conversion, based on a type mapping specified in the UI?

Reply:
yes it is for customization, updated statement